fearless/en.lproj/Localizable.strings

```diff
@@ -46,9 +46,9 @@ operations will appear here";
 "NSFaceIDUsageDescription" = "Face ID is used to authorize in application";
 "NSPhotoLibraryUsageDescription" = "Load photos from library";
 "NSPhotoLibraryAddUsageDescription" = "Save transfer request as a qr code";
-"username.setup.choose.title" = "Name";
-"username.setup.hint" = "This name will be displayed only for you and stored locally on your mobile device.";
-"username.setup.title" = "Create account";
+"username.setup.choose.title" = "Wallet nickname";
+"username.setup.hint" = "Example: Savings, Investments, Crowdloans, Staking. This nickname will be displayed only for you and stored locally on your mobile device.";
+"username.setup.title" = "Create a new wallet";
```

Comment:
@Viscount88  please add ru localization as well

Reply:
I created a separate task to localize the entire flow, so as not to request localization several times. https://soramitsu.atlassian.net/browse/FLW-1395